python/pyspark/sql/functions.py

```diff
@@ -2189,11 +2189,16 @@ def from_json(col, schema, options={}):
     >>> df = spark.createDataFrame(data, ("key", "value"))
     >>> df.select(from_json(df.value, schema).alias("json")).collect()
     [Row(json=[Row(a=1)])]
+    >>> schema = schema_of_json(lit('''{"a": 0}'''))
```

Comment:
nit: `'''{"a": 0}'''` -> `'{"a": 0}'`

Reply:
feel free to fix other examples above too

Reply:
Do you mean for other functions too?

Reply:
Nope, I mean the examples here in this function.